lib/common/crmcommon_private.h

```diff
@@ -276,7 +276,6 @@ pcmk__ipc_methods_t *pcmk__pacemakerd_api_methods(void);
  *
  * It is set when logging is initialized, and does not need to be set directly.
  */
-G_GNUC_INTERNAL
 extern bool pcmk__is_daemon;
```

Comment:
I didn't realize this was library-local, let's move this to include/crm/common/internal.h (or add an accessor function)

Reply:
Fixed.